Parser/StarmonMiniParse.m

```diff
@@ -314,7 +314,7 @@
     iContent = i + nColumnTime;
     
     switch header.param(iParam).column
-        case 'Temp(�C)'
+        case 'Temp0x280xB0C0x29' %degrees C
```

Comment:
My bad, the translation is correct :

```matlab
K>> genvarname('Temp(�C)')

ans =

Temp0x280xFFFDC0x29

K>> genvarname('Temp(°C)')

ans =

Temp0x280xB0C0x29
```

It's just that we're retrieving different strings when reading the file. For some reason you manage to read the '°' symbol properly while on my machine it is converted into '�'.